app/scripts/services/resourceAlerts.js

```diff
@@ -74,7 +74,8 @@ angular.module("openshiftConsole")
     var setQuotaNotifications = function(quotas, clusterQuotas, projectName) {
       var notifications = QuotaService.getQuotaNotifications(quotas, clusterQuotas, projectName);
       _.each(notifications, function(notification) {
-        if(!NotificationsService.isNotificationPermanentlyHidden(notification)) {
+        // Only add a quota notification if it wasn't specifically set to be hidden by user
+        if(!NotificationsService.isNotificationHidden(notification)) {
```

Comment:
I don't think this should be necessary since the drawer tracks what was cleared in session storage. I think the drawer should know whether to show or hide the notification already.

Reply:
> My worry is that this doesn't fix the problem for other notifications that don't include the project name in the ID.

That's why I was working in `namespace` into the common NotificationsService :smile: 
> 
> @dtaylor113 If you prepend the project name to the uid the notification drawer assigns, does it just work?
> 
> https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/notifications/notificationDrawerWrapper.js#L187
> 
> This might be a one line fix without needing the common changes?

I'll have to test after removing the `isNotificationHidden(..)` call in resourceAlerts.js.   The drawer currently uses EventsService.isCleared() for filtering the APIEvents; I need to add the `EventsService.isCleared()` call to the notifications callback.